### ISSUE TITLE
Looking for more training details of given models

### ISSUE THREAD

**Issue by infZhou** [phil-bergmann/tracking_wo_bnw#165]
Hi, thanks for your magical work! 
As I'm going to reproduce the result, I'd like to seek for more training details about the models provided (in the link https://vision.in.tum.de/webshare/u/meinhard/tracking_wo_bnw-output_v5.zip).  
(1) Are the default settings in /experiments/scripts/obj_det_training.py match the ones you used in training the model of "faster_rcnn_fpn_training_mot_17/model_epoch_27.model"? It seems you use a coco pretrained FRCNN-RPN model, an inital learning rate 0.001, and drop it by 10 times at the 20th epoch. A batch size of 4 is used, and the model is trained by 30 epoches in total. Besides, it seems the model was trained only on [MOT17-02, MOT17-05, MOT17-09, MOT17-11] four sequences, instead of all 7 MOT17 training sequences. 
(2) Whether was the ReID model (/reid/market1501/model/model-last.pth.tar) trained on MOT17 dataset? If so, on which split was it trained on?  It seems it was only trained on the Market1501 dataset. 
Am I right? Looking forward to your reply. Thanks a lot!
